platform/broadcom/docker-syncd-brcm/start.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
Reason for changing? Using `#!/usr/bin/env bash` will work even if the bash installation directory ever changes.